modules/virt-delete-vm-web.adoc

```diff
@@ -7,18 +7,16 @@
 
 = Deleting a virtual machine using the web console
 
-Deleting a virtual machine permanently removes it from the cluster. +
-[NOTE]
-====
-When you delete a virtual machine, the data volume it uses is automatically deleted.
-====
+Deleting a virtual machine permanently removes it from the cluster.
 
 .Procedure
 
 . In the {product-title} console, click *Virtualization* -> *VirtualMachines* from the side menu.
 
-. Click the Options menu {kebab} of the virtual machine that you want to delete and select *Delete*.
+. Click the Options menu {kebab} beside a virtual machine and select *Delete*.
```

Comment:
The IBM Style guide doesn't like [Kabob](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#k).

Reply:
You're joking, right?